src/compas_rhino/ui/__init__.py

```diff
@@ -25,6 +25,14 @@
 from .mouse import Mouse
 from .rui import Rui
 
+import warnings
+
+warnings.warn(
```

Comment:
@brgcode this one got merged and should also be deleted/commented out until `compas_ui` comes to life ;)

Reply:
okay, will do